docs/products/mysql/concepts/high-availability-mysql.md

```diff
@@ -0,0 +1,106 @@
+---
+title: High availability of Aiven for MySQL®
+sidebar_label: High availability
+---
+
+Aiven for MySQL® is available on a variety of plans, offering different levels of high availability. The selected plan defines the features available.
```

Comment:
I may be wrong, but I thought HA by definition didn't include single node services?